### PR TITLE
use head origin to find bads

### PR DESCRIPTION
**Pull request by agramfort**
WIP

- [x] Changelog has been updated (`docs/source/changes.md`)

### CODE REVIEW

**Comment by hoechenberger** [comment]
@agramfort Feel free to merge once it's working for you!

File: scripts/preprocessing/01-import_and_maxfilter.py [hunk 1]
```diff
@@ -150,6 +150,8 @@ def find_bad_channels(raw, subject, session, task, run):
         raw=raw,
         calibration=get_mf_cal_fname(subject, session),
         cross_talk=get_mf_ctc_fname(subject, session),
+        origin=config.mf_head_origin,
+        coord_frame='head',
```
Comment (hoechenberger) [inline scripts/preprocessing/01-import_and_maxfilter.py:154]: Good catch!!!

**Comment by hoechenberger** [comment]
Oh and would be great to have a tiny changelog entry in the bugfixes section 😁

**Review by hoechenberger — APPROVED** [review]

**Comment by hoechenberger** [comment]
Thanks @agramfort!

**Comment by agramfort** [comment]
Please merge when green

**Pull request merged**